README.md

```
@@ -1,2 +1,75 @@
+# Raytracer :white_circle: :sunny:
+Raytracer is a project we've done at Epitech. The goal of this project is simple : do a raytracer.
+
+## Setup 
+### 1. Install libconfig & SFML
+- On Ubuntu:
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines
null

(MD032, blanks-around-lists)